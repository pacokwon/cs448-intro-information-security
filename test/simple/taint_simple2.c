#include "homework.h"

int main() {
	int x = source(); 
	int y = sanitizer(x);
	int z = y * 100;
	int result;

	if(x > 0){
		result = x + y;
		sink(result);
	}
	else{
		result = y + z;
		sink(result);
	}
	return 0;
}
