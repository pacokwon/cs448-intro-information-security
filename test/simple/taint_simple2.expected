Potential Tainted-flow @ simple/taint_simple2.c:main:11:3 (call void @sink(i32 %add), !dbg !24)
